tests/test_mg_basic/results_true.dat

```diff
@@ -1,2 +1,2 @@
 k-combined:
-1.160653E+00 6.227102E-02
+5.788739E-01 1.945182E-02
```

Comment:
Hmm...not really. Let me dig into that a bit since this is a pretty big difference!